objc/message.h

```diff
@@ -52,8 +52,18 @@ id objc_msgSend(id self, SEL _cmd, ...);
 OBJC_PUBLIC
 #ifdef __cplusplus
 id objc_msgSend_stret(id self, SEL _cmd, ...);
+
+#	if defined(_WIN32) && defined(__ARM_ARCH_ISA_A64)
```

Comment:
An you add a doc comment explaining why this one exists?

Reply:
Sure!